### ISSUE TITLE
How to see what is read by read_drugbank_xml_db

### ISSUE THREAD

**Issue by iDemirsoy** [ropensci/dbparser#116]
Currently using this `dbparser` to read drugbank data. I wanted to check what is being read by the `read_drugbank_xml_db` function. I have tried ls() but it does not show anything. In Rstudio when I start spelling dru.. I can see they are showing like drug_pathways() but is there a function to show objects, columns or data list read by the `read_drugbank_xml_db`

**Comment by MohammedFCIS** [comment]
Hi @iDemirsoy 
* ```read_drugbank_xml_db``` is meant to read DrugBank database and load it in memory for the parser functions to work* 
* The data that is read is stored internally in dbparser package data structure and is not meant to be read by the user hence you will have nothing if you called this function only.
* This function needs to be called only once then you can call any parser function you need after that.
* Parser functions returns tibbles with the parsed data.
* For more information please visit https://docs.ropensci.org/dbparser/reference/read_drugbank_xml_db.html
* and for the returned parser structure please visit https://docs.ropensci.org/dbparser/reference/index.html
* you can also check that tutorial for a longer example https://docs.ropensci.org/dbparser/articles/dbparser.html
I hope that answers your question.
Please feel free to open if not or submit another issue for any question.
Thank you
